cloudwatch_logs_common/include/cloudwatch_logs_common/log_publisher.h

```diff
@@ -26,6 +26,18 @@
 
 namespace Aws {
 namespace CloudWatchLogs {
+  
+/** 
+ * @enum Aws::CloudWatchLogs::LogPublisherRunState
+ * @brief Defines the different runtime states for the Publisher
+ * This enum is used by the LogPublisher to track the current runtime state of the Run function
+ */
+enum LogPublisherRunState {
+  LOG_PUBLISHER_RUN_CREATE_GROUP,
+  LOG_PUBLISHER_RUN_CREATE_STREAM,
+  LOG_PUBLISHER_RUN_INIT_TOKEN,
+  LOG_PUBLISHER_RUN_CREATE_SEND_LOGS
```

Comment:
Seems like this should just be `LOG_PUBLISHER_RUN_SEND_LOGS`, as it's not creating any, and keeps it consistent with the function name.

Reply:
Good catch. Fixed.